go-controller/pkg/kubevirt/switch.go

```diff
@@ -0,0 +1,18 @@
+package kubevirt
+
+import (
+	"net"
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+const (
+	ARPProxyIPv4 = "169.254.1.1"
+	ARPProxyIPv6 = "d7b:6b4d:7b25:d22f::1"
+)
+
+func ARPProxyLSPOption() string {
```

Comment:
right; could you provide a docstring for this function ? It's return type is not immediately clear. 

Another question; IIUC, this function is returning all the addresses of the LSP. Shouldn't the func name feature the word "address" ? It's more explicit.

Reply:
> GetARPProxy is better I think
>

The returned format is the one expected at the `arp_proxy` LSP Option also I will use Compos since it's not reading it from a struct or anything, so `ComposeARPProxyLSPOption`
 
> What happens if v4 or v6 isnt enabled? Youre always returning it.

Did it to simplify logic, this will not break OVN or anything it just install some flows that answer ARP/NDP in case of ipv4 or ipv6 packages and they are not massive so the gain deactivating some of them is not worth it.

Reply:
> right; could you provide a docstring for this function ? It's return type is not immediately clear.
> 
> Another question; IIUC, this function is returning the MAC address of the LSP plus the ARP proxy IPs; shouldn't the func name feature the word "address" ? It's more explicit IMO.

Explaining the mac on docstring, name just state that this is the arp_proxy option for kubevirt.